core/src/unprocessed_transaction_storage.rs

```diff
@@ -728,14 +730,9 @@ impl ThreadLocalUnprocessedPackets {
                 })
                 .unzip();
 
-        // report metrics
         inc_new_counter_info!("banking_stage-packet_conversion", 1);
```

Comment:
just a note - this `inc` call is left for next round of cleaning?

Reply:
Correct - there are still a few other instances of `inc_new_counter` in `BankingStage` that can/should eventually be cleaned up.

But for the sake of keeping the PR focused and as easy to review as possible, I only swept up the ones that touched `banking_stage-dropped_tx_before_forwarding`.

Reply:
After our chat yesterday, I think the order of operations would be:
- Condense all remaining counters into existing datapoints (`BankingStageStats` or `LeaderSlotMetricsTracker`)
- From there, start removing duplicate fields between `BankingStageStats` and `LeaderSlotMetricsTracker`

As you pointed out, `BankingStageStats` may not fully go away as we may be interested to see some data for things that don't necessarily happen during the leader slots. Like maybe the flow of packets before/after the node's actual leaders slots